pandas/core/ops/__init__.py

```diff
@@ -635,6 +624,25 @@ def _frame_arith_method_with_reindex(
     return result.reindex(join_columns, axis=1)
 
 
+def _align_series_as_frame(frame: "DataFrame", series: "Series", axis: int):
```

Comment:
```suggestion
def _maybe_align_series_as_frame(frame: "DataFrame", series: "Series", axis: int):
```

(since it doesn't always do this?)

Reply:
will do